src/LodeRunner.API.Test/IntegrationTests/TestPayloads/TestRunTestPayload.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using LodeRunner.API.Test.IntegrationTests.TestPayloads;
+using LodeRunner.Core.Models;
+
+namespace LodeRunner.API.Test.IntegrationTests.Payloads
+{
+#pragma warning disable SA1600 // Elements should be documented
+    public class TestRunTestPayload : BaseTestPayload

```

Comment:
Is this the best way to do the test payload?  I believe there's a way to put the entity into files and specify it in the attribute so that it uses the file content to create a new object.  If we can do that then we could create new payloads without modifying code and without disabling a warning.

Please look into it and share what you find out.

Reply:
Used TestRunPayload